src/seismometer/controls/thresholds.py

```diff
@@ -0,0 +1,148 @@
+import logging
+from collections.abc import Iterable
+
+import traitlets
+from ipywidgets import FloatSlider, Layout, ValueWidget, VBox
+
+from seismometer.plot.mpl._ux import alert_colors
+
+from .styles import WIDE_LABEL_STYLE
+
+logger = logging.getLogger("seismometer")
+
+
+class PercentSliderListWidget(ValueWidget, VBox):
+    """
+    Vertical list of sliders
```

Comment:
Can you add to the docstring that these are 0-1 in increments of 0.01.  
Two things that I want to make explicit. The bounds (-250% is well defined, I think, but not allowed) and that your range are NOT the integer percentiles (0-100).  

Elsewhere for logit 'scores' I believe we have used 'percentiles' as 0-100 and 'probabilities' as 0-1.

Reply:
Will do, also changing the name to ProbabilitySliderListWidget. I'll write up an issue for controls needing to support non 0-1 thresholds and needing threshold information in config.yml to allow named threshold and threshold limits in the future.